includes/class-sensei-course-structure.php

```diff
@@ -689,4 +689,34 @@ private function validate_item_structure( array $raw_item ) {
 
 		return true;
 	}
+
+	/**
+	 * Sort structure.
+	 *
+	 * @param array $structure     Structure to be sorted.
+	 * @param int[] $lessons_order Order to sort.
+	 *
+	 * @return array Sorted structure.
+	 */
+	public static function sort_structure( $structure, $lessons_order ) {
```

Comment:
The tests for this method will be written in a next PR for the modules sorting. It'll also be refactored there to make it useful for both types.